stdlib/2and3/xml/etree/ElementTree.pyi

```diff
@@ -10,53 +10,80 @@ class ParseError(SyntaxError): ...
 
 def iselement(element: 'Element') -> bool: ...
 
-_Ss = TypeVar('_Ss', str, bytes)
 _T = TypeVar('_T')
-_str_or_bytes = Union[str, bytes]
+
+# Type for parser inputs. Parser will accept any unicode/str/bytes and coerce,
+# and this is true in py2 and py3 (even fromstringlist() in python3 can be
+# called with a homogeneous list)
```

Comment:
Do you mean heterogeneous? (i.e., the function accepts a list containing both bytes and str in py3)